client/reader/stream/empty-search-recommended-post.jsx

```diff
@@ -26,10 +26,11 @@ export default function EmptySearchRecommendedPost( { post } ) {
 		recordAction( 'search_page_rec_site_click' );
 	}
 
-	const site = { title: post.site_name };
+	const site = { title: post && post.site_name };
 
+	/* eslint-disable  wpcalypso/jsx-classname-namespace */
 	return (
-		<div className="search-stream__recommendation-list-item" key={ post.global_ID }>
+		<div className="search-stream__recommendation-list-item" key={ post && post.global_ID }>
```

Comment:
if there's no post, there's no key... 

should the parent be assigning the key?